content/600-about/200-prisma-docs/20-style-guide/05-prisma-product-names.mdx

```diff
@@ -5,17 +5,48 @@ metaDescription: 'This section of the style guide provides guidelines on how to
 tocDepth: 3
 ---
 
-## Use the following forms
+Use the official Prisma product names and feature names listed below in documentation. Do not abbreviate any of these names, except where noted.
+
+## Product names
 
 - The Prisma Data Platform (or just "PDP" - no definite article - after the first mention)
 - Prisma Client (not any variation of "the Client")
 - Prisma Schema
 - Prisma Migrate
 - Prisma CLI
 - Prisma Studio
+
+## Feature names
+
 - Data Proxy (_not_ Prisma Data Proxy)
 - Data Browser (in the Prisma Data Platform UI)
 - Query Console (in the Prisma Data Platform UI)
 - Schema Viewer (in the Prisma Data Platform UI)
 
-Do not abbreviate any of these names, except where noted above.
+## Use of articles with product names and feature names
+
+Do not use a definite article in front of official product names.
+
+```
+<!-- Good -->
+
+You use Prisma Client to send queries to the database.
+
+<!-- Bad -->
+
+You use the Prisma Client to send queries to the database.
+
+```
+
+Where appropriate, you can use definite articles in front of feature names.
+
+```
+<!-- Good -->
+
+The Data Proxy provides database connection management and pooling.
```

Comment:
It seems that Prisma uses the definite article for the Data Proxy - e.g. on our [home page](https://www.prisma.io/)

When we have the results of Julie's investigation into this, we can update the list above.